frontend-react/src/components/Admin/EditSenderSettings.tsx

```diff
@@ -253,39 +254,31 @@ const EditSenderSettingsForm: React.FC<EditSenderSettingsFormProps> = ({
 };
 
 export type EditSenderSettingsProps = {
-    orgName: string;
-    senderName: string;
+    orgname: string;
+    sendername: string;
```

Comment:
Changed back because they're specific to the URL string. We can clean this up in a subsequent PR if camel case is mega important!